src/core/interp.c

```diff
@@ -37,9 +37,18 @@ MVM_STATIC_INLINE MVMuint16 check_lex(MVMThreadContext *tc, MVMFrame *f, MVMuint
 #endif
 #define GET_I16(pc, idx)    *((MVMint16 *)(pc + idx))
 #define GET_UI16(pc, idx)   *((MVMuint16 *)(pc + idx))
-#define GET_I32(pc, idx)    *((MVMint32 *)(pc + idx))
-#define GET_UI32(pc, idx)   *((MVMuint32 *)(pc + idx))
-#define GET_N32(pc, idx)    *((MVMnum32 *)(pc + idx))
+
+MVM_STATIC_INLINE MVMint32 GET_I32(const MVMuint8 *pc, MVMint32 idx) {
+    MVMint32 retval;
+    memcpy(&retval, pc + idx, sizeof(retval));
+    return retval;
+}
+
+MVM_STATIC_INLINE MVMuint32 GET_UI32(const MVMuint8 *pc, MVMint32 idx) {
+    MVMuint32 retval;
+    memcpy(&retval, pc + idx, sizeof(retval));
+    return retval;
+}
```

Comment:
Does this have performance consequences on platforms that don't need it, or are compilers good enough at optimizing the `memcpy` out? Doesn't really matter on the other code paths, but the interpreter is a bit more sensitive.

Reply:
I think that this is answered by the last paragraph of commit 87ad486f30ecdf28b4abbcf40139467f626f8548

Reply:
Oh, except that IIRC the code I was looking at was the assembler output from interp.c, and it doesn't say that \:-)

Reply:
Ah, I somehow missed that comment. Thanks.